src/ConnectionManager.js

```diff
@@ -69,6 +71,10 @@ define(["structures/Response", "structures/Request", "structures/CAPIError"],
             }
         }
 
+        if (null !== this._siteAccess) {
```

Comment:
actually when the siteaccess is not provided `this._siteAccess` is undefined and I think we don't have to be super precise in that condition, so:
```js
if ( this._siteAccess ) {
```
would be enough